authentication/plaid_middleware.py

```diff
@@ -8,7 +8,7 @@
 PLAID_CLIENT_ID = os.environ.get('PLAID_CLIENT_ID')
 PLAID_SECRET = os.environ.get('PLAID_SECRET')
 PLAID_PUBLIC_KEY = os.environ.get('PLAID_PUBLIC_KEY')
-PLAID_ENV = 'sandbox' if os.environ.get('DEBUG') == "TRUE" else 'development'
+PLAID_ENV = 'sandbox'
```

Comment:
why you do dis

Reply:
Cuz Travis doesn't set DEBUG to true. Going to use a separate env variable.